admin/controllers/pupil.js

```diff
@@ -88,7 +85,7 @@ controller.getAddMultiplePupils = (req, res, next) => {
 controller.postAddMultiplePupils = async (req, res, next) => {
   let school
   try {
-    school = await School.findOne({_id: req.user.School}).exec()
+    school = await schoolDataService.findOne({_id: req.user.School})
```

Comment:
how about adding a method to pupil service for now that all it does is call school and pupil data service while being wrapped in a try catch?

Reply:
That's a good call but outside the intended scope of this PR - and there are 8 PRs to tackle just to move all mongoose models into the data layer.